modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/util/MuleExtensionUtils.java

```diff
@@ -410,6 +445,11 @@ public static IllegalModelDefinitionException noClassLoaderException(String exte
         return new IllegalModelDefinitionException("No ClassLoader was specified for extension " + extensionName);
     }
 
+    private static boolean isConnected(EnrichableModel o)
```

Comment:
Why do you move methods in the same class? There seems to be no reason, but it just adds lines to the change, which is already pretty big